earthly/rust/tools/Earthfile

```
@@ -0,0 +1,57 @@
+VERSION 0.8
+
+tool-cargo-nextest:
+    FROM ../+rust-base
+    RUN cargo install cargo-nextest --version=0.9.68 --locked
+    SAVE ARTIFACT /usr/local/cargo/bin/cargo-nextest cargo-nextest
```

According to Earthly reference:

> If `<artifact-dest-path>` is not specified, it is inferred as `/`

So the below suggestion should also work, without specifying name twice.

```suggestion
    SAVE ARTIFACT /usr/local/cargo/bin/cargo-nextest
```